src/Datadog.Trace.ClrProfiler.Managed/AutoInstrumentation/Testing/NUnit/NUnitIntegration.cs

```diff
@@ -75,6 +75,7 @@ internal static Scope CreateScope<TContext>(TContext executionContext, Type targ
             // Get traits
             if (testMethodProperties != null)
             {
+                Dictionary<string, List<string>> traits = new Dictionary<string, List<string>>();
```

Comment:
We really should use var in those cases 😄

Reply:
oh, not again... the `var` discussion...

Reply:
If you feel like a different argument, you could use a [new expression](https://docs.microsoft.com/en-us/dotnet/csharp/language-reference/operators/new-operator) instead :trollface: